api/app.ts

```diff
@@ -19,6 +19,10 @@ app.use(
     })
 )
 
+// POST endpoint for submiting surveys
+import surveyEndpoint from './survey'
```

Comment:
End lines in ts with Semicolon
```suggestion
import surveyEndpoint from './survey';
```